projects/plugins/protect/src/js/components/firewall-page/styles.module.scss

```diff
@@ -56,3 +61,38 @@
 .go-back-button {
     margin: calc( var( --spacing-base ) * -1 ) calc( var( --spacing-base ) * -2 ); // -8px | -16px
 }
+.badge {
+    color: var( --jp-gray-30 );
+    font-size: 12px;
+    height: 21px;
+    padding: calc( var( --spacing-base ) / 2 ); // 4px
+    border: 1px solid var( --jp-gray-30 );
+    border-radius: 4px;
+    margin-left: var( --spacing-base ); // 8px
+    margin-bottom: calc( var( --spacing-base ) * 2 ); // 16px
+    display: flex;
+    align-items: center;
+}
+
+.popover {
+    width: 304px;
+    padding: calc( var( --spacing-base ) * 3 ); // 24px
+
+    > .popover-header {
```

Comment:
Super minor, I've been using [BEM](https://getbem.com/) naming convention in this file. To convert this to BEM, we might name it `.popover__header` instead.

You can nest it like so as well:
```scss
.popover {
    // ...

    &__header {
        // ...
    }
}

With the above approach, the popover child elements are still nested in the `.popover` class, but the resulting CSS is actually more like this:

```css
.popover { /* ... */ }
.popover__header { /* ... */ }
```

...which keeps all of the classes with the same specificity, which has [some benefits](https://getbem.com/faq/).